tests/programs/custom/interrupt_test/interrupt_test.h

```diff
@@ -14,6 +14,8 @@
 // Enable debug messages, note that this will change test timing
 //#define DEBUG_MSG
 
+#define EXIT_SUCCESS   0
```

Comment:
I think it could be better to include [stdlib.h](https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/basedefs/stdlib.h.html).

Reply:
But its... pretty much the same so it should be fine like it is now too.